source/_posts/2017-09-01-gsoc-2017-data-visualization-using-daru-view.markdown

```diff
@@ -0,0 +1,284 @@
+---
+layout: post
+title: "GSoC 2017 : Data visualization using daru-view"
+date: 2017-09-01 22:46
+comments: true
+author: Shekhar Prasad Rajak
+categories: [GSoC,GSoC 2017,Students,Data Visualization,daru,daru-view,
+  Charts,Plotting,Data Analysis]
+---
+
+Hello, Rubyist! Are looking for good ruby gem for interactive data
+visualisation? Then you must try [`daru-view`](https://github.com/Shekharrajak/daru-view), a plugin gem for [`daru`](https://github.com/SciRuby/daru).
+
+## What makes daru-view different ?
+
+* `daru-view` is desinged for interactive plotting of charts and tables.It
+provide different plotting tools like Nyaplot, HighCharts, GoogleCharts,
+DataTable. So you don't have to write any JavaScript code from these sites
+and no need to shift to other language to get charts.
+
+* It can work with **any ruby web application framework like Rails/Sinatra/Nanoc**. If you want to try few examples then please look into the
+`daru-view/spec/dummy_*` examples of Rails, Sinatra, Nanoc web applications.
+
+* Now Ruby developers are using IRuby notebook for few lines of code.
+`daru-view` support IRuby notebook as well. So if you just want to see chart
+for some DataFrame or Array of data, you can use `daru-view`.
+
+* `daru-view` can generate chart images to download and save.
+
+
+## Introduction
+
+Daru is doing pretty good work as the data analysis & manipulation in IRuby notebook as well as backend part of web application. Ruby web application frameworks like Ruby on Rails, Sinatra, Nanoc are popular frameworks. So if Ruby developers get the gem like daru which can do data analysis
+and visualization work in applications, then there is no need of shifting to another language or usage of other gem.
+
+My project for <abbr title="Google Summer of Code 2017"> GSoC 2017</abbr> was to "make Daru more ready for integration with modern Web framework" in terms of visualization.
+
+To improve in terms of viewing data,
+[daru-view](https://github.com/Shekharrajak/daru-view), a plugin gem for
+[daru](https://github.com/SciRuby/daru) is created. [daru-view](https://github.com/Shekharrajak/daru-view) is for easy and interactive plotting in web application & IRuby notebook. It can work in frameworks like Rails, Sinatra, Nanoc and hopefully in others too.
+
+If you just want to see the features of daru-view, you can check these
+examples :
+
+* [IRuby Notebook Examples](http://nbviewer.jupyter.org/github/shekharrajak/daru-view/tree/master/spec/dummy_iruby/)
+
+* [daru io and daru-view usage in Rails app](https://github.com/Shekharrajak/daru_examples_io_view_rails)
+
+* [Readme of daru-view](https://github.com/Shekharrajak/daru-view)
```

Comment:
README should be all caps and linking to the README.md file not the full repo. I would put the README as the